bin/pycbc_live

```diff
@@ -968,6 +968,9 @@ parser.add_argument('--gracedb-labels', metavar='LABEL', nargs='+',
 parser.add_argument('--ifar-upload-threshold', type=float, required=True,
                     help='Inverse-FAR threshold for uploading candidate '
                          'triggers to GraceDB, in years.')
+parser.add_argument('--coinc-threshold', type=float,
+                    help="The time window allowed between coincident triggers",
+                    default=0.002)
```

Comment:
This must have had an at least implied value before - how was that set?

Reply:
Very very good question, the only other place I can remember off the top of my head is here: [link](https://github.com/gwastro/pycbc/blob/master/pycbc/events/coinc.py#L241) but I don't know if this is exactly where it is use for the time window for coincidences in the other ranking statistics.

Reply:
Seems like you are adding this option but not using it anywhere, is this intended? Or is it being picked up "indirectly" within some deeper module?

Reply:
It is fed through in the `args` to `coinc.py` on [L1130](https://github.com/ArthurTolley/pycbc/blob/template_fits/bin/pycbc_live#L1130) to fulfil the class argument: [L968](https://github.com/ArthurTolley/pycbc/blob/template_fits/pycbc/events/coinc.py#L968) (in coinc.py).

Reply:
The help string is not correct. The docstring of the class claims that `coinc_threshold` is the

> Amount of time allowed to form a coincidence in addition to the time of flight in seconds.

I would rather call this quantity (and the CLI option) `coinc_window_pad`, `coinc_window_extra` or something like that. Maybe first check with Alex where the word "threshold" came from though.

Reply:
`coinc_window_pad` definitely makes more sense if this quantity is in addition to the
 light travel time. I will happily change this unless @ahnitz has an objection?